### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get' when executing job

### ISSUE THREAD

**Issue by knjk04** [databrickslabs/dbx#282]
## Expected Behavior

Successfully executes job on the interactive cluster.

## Current Behavior

```AttributeError: 'NoneType' object has no attribute 'get'```

Stack trace:

```
> dbx execute --job=dbx-demo-sample-multitask --cluster-name=git-proxy --deployment-file=conf/deployment2.yml
[dbx][2022-06-10 14:27:50.262] No environment variables provided, using the ~/.databrickscfg
[dbx][2022-06-10 14:27:50.921] Executing job: dbx-demo-sample-multitask in environment default on cluster git-proxy (id: 0516-145600-2xwplki1)
[dbx][2022-06-10 14:27:50.921] Re-building package
/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
[dbx][2022-06-10 14:27:52.478] Package re-build finished
[dbx][2022-06-10 14:27:52.478] Using the provided deployment file conf/deployment2.yml
Traceback (most recent call last):
  File "/Users/karankumar/.virtualenvs/dbx/bin/dbx", line 8, in <module>
    sys.exit(cli())
  File "/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/karankumar/.virtualenvs/dbx/lib/python3.8/site-packages/dbx/commands/execute.py", line 104, in execute
    entrypoint_file = job_payload.get("spark_python_task").get("python_file")
AttributeError: 'NoneType' object has no attribute 'get'
```

## Steps to Reproduce (for bugs)

1. Create a sample project: `dbx init --template=python_basic`

`conf/deployment.yml`:

```yaml
 # Custom section is used to store configurations that might be repetative.
# Please read YAML documentation for details on how to use substitutions and anchors.
custom:
  basic-cluster-props: &basic-cluster-props
    spark_version: "9.1.x-cpu-ml-scala2.12"

  basic-static-cluster: &basic-static-cluster
    new_cluster:
      <<: *basic-cluster-props
      num_workers: 1
      node_type_id: "i3.xlarge"

# please note that we're using FUSE reference for config file, hence we're going to load this file using its local FS path
environments:
  default:
    strict_path_adjustment_policy: true
    jobs:
      - name: "dbx-demo-sample-multitask"
        job_clusters:
          - job_cluster_key: "default"
            <<: *basic-static-cluster
        tasks:
          - task_key: "etl"
            job_cluster_key: "default"
            python_wheel_task:
              package_name: "dbx_demo"
              entry_point: "etl"
              parameters: [ "--conf-file", "file:fuse://conf/test/sample_etl_config.yml" ]
          - task_key: "ml"
            depends_on:
              - task_key: "etl"
            job_cluster_key: "default"
            python_wheel_task:
              package_name: "dbx_demo"
              entry_point: "ml"
              parameters: [ "--conf-file", "file:fuse://conf/test/sample_ml_config.yml" ]
          ###############################################################################
          # this is an example task based on the notebook                               #
          # Please note that first you'll need to add a Repo and commit notebook to it. #
          ###############################################################################
          - task_key: "notebook"
            deployment_config:
              no_package: true # we omit using package since code will be shipped directly from the Repo
            depends_on:
              - task_key: "ml"
            job_cluster_key: "default"
            notebook_task:
              notebook_path: "/Repos/[email-address-omitted]/dbx_demo/sample_notebook"
```

I have created the repo and added the sample notebook to it. Nothing stands out when I run the `execute` command with the debug flag. I got 200 OK responses.

## Your Environment

* dbx version used: 0.5.0
* Databricks Runtime version: 0.16.6

**Comment by knjk04** [comment]
Thanks, @renardeinside. 

Would it be possible to document that this is not yet supported in dbx in this guide: https://dbx.readthedocs.io/en/latest/multitask_jobs.html?

Also, I ran `dbx init --template=python_basic` and got the template in my original post. Would it be worth not generating the  multitask job configuration (or commenting out the multitask jobs section and explain with a comment that it is not yet supported)? 

I think both suggestions may help to avoid confusion with others.

**Comment by seanlindo** [comment]
I spent a decent amount of time going around in circles before discovering this. I also used `dbx init` and immediately went to executing multitask jobs after getting through a basic POC. I have to imagine this is not an unrealistic path.

**Comment by mickvanhulst-TomTom** [comment]
Just to clarify, the template that we obtain when running `dbx init --template=python_basic` does not result in a template that I can actually complete the "Quickstart" tutorial with?

@knjk04 This is also what you've experienced?

Is there an expectation when this will likely be released?

**Comment by renardeinside** [comment]
Addressed in the linked PR, will be released as a part of 0.6.6
